src/components/footers/OutputBox.tsx

```diff
@@ -208,6 +210,9 @@ export const OutputBox = () => {
   const toggleCommentsDetail = useCallback(() => {
     setIsShowDetails(!isShowDetails);
   }, [isShowDetails]);
+  const toggleLoadJSON = useCallback(() => {
+    setIsShowLoadJSON(!isShowLoadJSON);
+  }, [isShowLoadJSON]);
```

Comment:
もともとの書き方が悪いのですが、setStateは以下のように実装したほうがよいです

参考: https://ja.react.dev/reference/react/useState#updating-state-based-on-the-previous-state
```suggestion
  const toggleLoadJSON = useCallback(() => {
    setIsShowLoadJSON(pv=>!pv);
  }, []);
```